doc/source/dev/toolchain.rst

```diff
@@ -254,7 +229,6 @@ OpenBLAS, ATLAS, MKL, BLIS, and reference Netlib libraries are known to work.
 =============== =====================================================
 LAPACK           3.4.1
```

Comment:
I doubt that support is still functional that far back (3.4.1 was [released](https://github.com/Reference-LAPACK/lapack#lapack) in April 2012), but I couldn't quickly find out what a reasonable (and believably tested!) minimum would be.

Reply:
The point is that the APIs in 3.4.1 are required, for newer LAPACK versions MKL is missing them.

Reply:
MKL supports up to LAPACK 3.9.0 (since MKL 2020.4), see e.g. [here](https://github.com/conda-forge/intel_repack-feedstock/issues/6). There are more issues tracking support (for 3.9.1 & 3.10.0) on that feedstock.